tls/s2n_connection.c

```diff
@@ -52,6 +52,9 @@
 #include "utils/s2n_socket.h"
 #include "utils/s2n_timer.h"
 
+#define S2N_GENERATE_EMPTY_KEY_SHARE_LIST(keyshares) (keyshares |= 1)
+#define S2N_GENERATE_KEY_SHARE_FOR_SELECTED_GROUP(keyshares, i) (keyshares |= ( 1 << ( i + 1 )))
```

Comment:
Same comment as above, but it's even more confusing because these macros have the same names but do different things.

How about "set_key_share_list_empty" and "set_key_share_request"?